README.md

```diff
@@ -6,7 +6,26 @@ They will be in `_build/html`
 ```bash
 make html
 ```
+
+
+
+If new files have been created in the source code repositories (such as lava, lava-on-loihi, lava-optimization), sphinx-apidoc must be re-run.
+
+
+```bash
+sphinx-apidoc -o <output_path> <path-to-src-repo>/src/lava -t _templates -H -d 10 --implicit-namespace
+```
```

Comment:
Apparently we need to put something after -H, to be used as prefix on the generated files. The actual word is probably irrelevant for rendering. But the command above will fail due to this missing argument.

Reply:
I removed the -H from the command. It's really not necessary and as you said faulty in this form.